en/docs/get-started/about-this-release.md

```diff
@@ -1,63 +1,229 @@
 # About this Release
 
-WSO2 API Manager is a platform for creating, managing, consuming, and monitoring APIs. It employs proven SOA best practices to solve a wide range of API management challenges such as API provisioning, API governance, API security, and API monitoring. It combines some of the most powerful and mature components of the WSO2's state-of-the-art Carbon platform to deliver a smooth and end-to-end API management experience while catering to both API publisher and API consumer requirements.
+The WSO2 API Manager is a platform for building and managing APIs. It helps developers implement APIs by integrating different types of systems and APIs and to manage them using its API governance and lifecycle management capabilities. APIs could be built by integrating different types of APIs such as traditional request-response style APIs and event-driven APIs using a low-code/no-code development experience and expose to consumers through its API Gateway. 
 
-WSO2 API Manager comprises of the following modules:
+For more information on WSO2 API Manager, see the [overview in the WSO2 API Manager 4.0.0 documentation]({{base_path}}/getting-started/overview/).
 
-- **API Publisher**: Allows API creators to design, implement, and document APIs and allows API product managers to manage the lifecycle of APIs and create API products by using one or more APIs.
+## What is new in this release
 
-- **Developer Portal**: A portal for onboarding application creators. It allows application creators to discover, subscribe, test and consume APIs through their applications.
+The **WSO2 API Manager 4.0.0-Alpha** is the **latest** **WSO2 API Manager release** and is the successor of **WSO2 API Manager 3.2.0**. It contains the following components, which have new features and enhancements:
```

Comment:
shouldn't we say this is the successor of EI as well?

Reply:
Will check on whether we should say that.